components/modals/EditTaskModal.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import moment from 'moment';
+import { NotificationDateTimeType, TaskData, TaskFormData, TaskPatchData } from '../../types/task';
+import { isValidDate } from '../../utils/date/validations';
+import { calculateDaysOffset } from '../../utils/date/formatters';
+import { Nullable } from '../../types/utils';
+import TaskForm from '../forms/TaskForm';
+import tasksApi from '../../api/tasksApi';
+import {
+  DEFAULT_NOTIFICATION_DAYS_OFFSET,
+  DEFAULT_NOTIFICATION_TIME,
+  getNotificationDateTime,
+} from '../../utils/task/taskUtils';
+import { Button, Modal } from 'antd';
+import { ExclamationCircleOutlined } from '@ant-design/icons';
+
+const { confirm } = Modal;
```

Comment:
You can just do `Modal.confirm`. All these destructing makes it seem like confirm is a function that you created.